content/components/segmented-control.mdx

```diff
@@ -0,0 +1,426 @@
+  </Dont>
+</DoDontContainer>
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://user-images.githubusercontent.com/2313998/180623815-d72a2c84-8353-4b84-965a-abe6ce1da724.png"
+      role="presentation"
+    />
+    <Caption>
+      Use a segmented control for selecting from a list of static values. You may use an auxiliary dropdown adjacent to
+      the segmented control.
+    </Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://user-images.githubusercontent.com/2313998/180624829-d10c1324-378c-435c-a39b-c41b1ffb8081.png"
+      role="presentation"
+    />
+    <Caption>Don't use a segmented control with a choice that is a dropdown.</Caption>
+  </Dont>
+</DoDontContainer>
+
+### Button icons
+
+<Box
+  display="flex"
+  alignItems="flex-start"
+  flexDirection={['column', 'column', 'column', 'column', 'row']}
+  sx={{gap: 3}}
+>
+  <Text as="p" mt="0">
+    An icon can be rendered in a segmented control button as a visual cue to help users understand the choice that
+    button represents.
+  </Text>
+  <img
+    src="https://user-images.githubusercontent.com/2313998/180623824-41ac7a02-0895-40f7-a973-0aa055a9503c.png"
+    role="presentation"
+    width="456"
+  />
+</Box>
+
+#### Effective visual cues
+
+Be consistent with the way visual cues are used in each button of the segmented control. Use either all icons, all text labels, or all text labels with icons.
+
+Do not change icons to communicate state.
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://user-images.githubusercontent.com/2313998/180624836-920ff9c3-ba88-4137-8d9d-6f4d06271e08.png"
+      role="presentation"
+    />
+    <Caption>Use icons to as a visual cue to communicate what the choice does.</Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://user-images.githubusercontent.com/2313998/180624838-1915681f-1c50-4d0e-94e9-98f3537cf285.png"
+      role="presentation"
+    />
+    <Caption>Change or add icons to indicate whether the choice is selected.</Caption>
+  </Dont>
+</DoDontContainer>
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://user-images.githubusercontent.com/2313998/180624832-b9ce99d7-e4e3-46fb-8b24-76133fc6dd90.png"
+      role="presentation"
+    />
+    <Caption>Keep the visual cues in each button consistent.</Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://user-images.githubusercontent.com/2313998/180624835-fc3c7c3b-9f55-4001-9e16-9c7e2071bf8d.png"
+      role="presentation"
+    />
+    <Caption>Don't mix visual cues between buttons.</Caption>
+  </Dont>
+</DoDontContainer>
+
+#### Icon-only buttons
+
+Use icons without a visible text label sparingly, and only with icons that are easily understood by an average GitHub user. When a segment has an icon without a visible text label, the text label should be shown in a tooltip.
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://user-images.githubusercontent.com/2313998/180624830-b91217ae-7c79-4698-a71a-a482576e5df9.png"
```

Comment:
The hover style shown in this image is looking a little squished. Is that also in the implementation?

Reply:
Yes, it's a 32x32 square with a 44px high click area on devices with a 
"coarse" pointer: https://primer.style/react/storybook/?path=/story/segmentedcontrol-examples--icons-only